data/migrations/20191108110543_bootstrap.js

```diff
@@ -0,0 +1,14 @@
+//creating projects table initially
+exports.up = function (knex) {
+	return knex.schema.createTable('projects', tbl => {
+		tbl.increments()
+		tbl.varchar('name', 255)
+			.notNullable();
+		tbl.varchar('description', 255);
+		tbl.boolean("complete").defaultTo(false);
```

Comment:
Don't forget that this column is also `.notNullable()`.

If someone tried to give a req.body like
```
{
    name: "New project",
    description: "blargh",
    completed: null
}
```
I'm... not sure if that would go through, but it might override the "false" default in the database. Always good to add the `.notNullable()` just in case 👍